src/command.hpp

```diff
@@ -153,6 +155,12 @@ namespace zmq
             struct {
             } term_ack;
 
+            //  Sent by session_base (I/O thread) to socket (application thread)
+            //  to ask to disconnect the endpoint.
+            struct {
+                std::string *endpoint;
```

Comment:
I think this should simply be a std::string, and maybe not a pointer to a string.
It seem to me it  is very unclear who is responsible for this pointer

Reply:
It must be either a pointer or a basic type, as it is part of a union